.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+install:
+- npm install
+- npm run bower:install
```

Comment:
It requires me to run `node_modules/.bin/bower install`. This is in my opinion less pleasing to the eye, making more assumptions about paths and less npm-style.